component/src/main/java/org/cbioportal/genome_nexus/util/GenomicVariantUtil.java

```diff
@@ -248,15 +248,16 @@ private static String getRefFromHgvs(String hgvs, GenomicVariant.Type type) {
                 }
                 return s.trim();
             case INSERTION:
-                return "";
+                return "-";
```

Comment:
Simply replaced empty strings with `-`, not sure if this would break something else